src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyDefendantJudgmentVariedDeterminationOfMeansNotificationHandler.java

```diff
@@ -29,6 +29,8 @@ public class NotifyDefendantJudgmentVariedDeterminationOfMeansNotificationHandle
 
     private static final List<CaseEvent> EVENTS = List.of(NOTIFY_SOLICITOR1_DEFENDANT_JUDGMENT_VARIED_DETERMINATION_OF_MEANS,
                                                            NOTIFY_SOLICITOR2_DEFENDANT_JUDGMENT_VARIED_DETERMINATION_OF_MEANS);
+
+    public static final String TASK_ID = "NotifyDefendantJudgmentVariedDeterminationOfMeans";
```

Comment:
Is TASK_ID used anywhere in this class?

Reply:
No, this one is not used so it was deleted